src/main/java/bio/terra/cloudres/util/MetricsHelper.java

```diff
@@ -0,0 +1,160 @@
+package bio.terra.cloudres.util;
+
+import bio.terra.cloudres.common.CloudOperation;
+import com.google.common.annotations.VisibleForTesting;
+import io.opencensus.common.Scope;
+import io.opencensus.stats.*;
+import io.opencensus.tags.*;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.OptionalInt;
+
+/** Util class to hold common variable and method used by OpenCensus in tracing and stats. */
+public class MetricsHelper {
+  public static final String CLOUD_RESOURCE_PREFIX = "terra/cloudresourcelibrary";
+  public static final ViewManager viewManager = Stats.getViewManager();
+
+  /**
+   * Fake HTTP status code value for errors that are not HTTP status errors. Useful for including
+   * non-HTTP status errors in a single metric.
+   */
+  @VisibleForTesting static final int GENERIC_UNKNOWN_ERROR_CODE = 1;
```

Comment:
```suggestion
  @VisibleForTesting static final int GENERIC_UNKNOWN_ERROR_CODE = 999;
```
could we use something more unique for ease of search?

Reply:
Will suggested to use 1 or 0. And I feel 1 is already unique enough for the purpose of http status code. 999 do sounds wired if that appears in our metrics to indicate an error..